build/container/external/packages/boost.cmake

```diff
@@ -0,0 +1,62 @@
+
```

Comment:
are these external package cmake files largely untouched from the build system of their original repo?  or did you have to author a lot of new logic for them?